app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from "react";
+
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import ContactsList from "./ContactsList";
+import ContactsMenuBar from "./ContactsMenuBar";
+
+const Contacts = () => {
+  const [showMenu, setshowMenu] = useState(true);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  useEffect(() => {
+    fetchContacts();
+  }, []);
+
+  const fetchContacts = () => {};
```

Comment:
Is this useEffect required? Does it add any value to the current page?

Reply:
Yes removed. thanks